### PR TITLE
Updated dwd examples

### PR DESCRIPTION
**Pull request by jsieland**
Hi everyone,

I updated the examples from DWD due to changes @antje-s and I made:

- update dwd-icon example according to recent changes in spec
- added dwd-global-cache draft example
- deleted dwd-metar example because deprecated and not a good example

As always, feedback is greatly appreciated.

Thanks,
Julia

### CODE REVIEW

File: examples/de.dwd.globalcache.json [hunk 1]
```diff
@@ -0,0 +1,128 @@
+{
+    "id": "urn:x-wmo:md:deu:dwd-offenbach:global-cache-service",
+    "type": "Feature",
+    "conformsTo": [
+        "http://wis.wmo.int/spec/wcmp/2.0"
+    ],
+    "time": {
```
Comment (tomkralidis) [inline examples/de.dwd.globalcache.json:7]: Thinking out loud, do/should we specify a time for a service, or do we advocate for `null` as a rule?

File: examples/wcmp2_de.dwd.icon-eps.ALL.json [hunk 2]
```diff
@@ -20,126 +20,111 @@
...
-            },
-            {
-                "rel": "items",
-                "type": "application/json",
-                "title": "Message Broker DWD",
-                "href": "mqtts://everyone:everyone@wis2.dwd.de",
-                "channel": "origin/a/wis2/deu/dwd-offenbach/data/core/weather/analysis-prediction/forecast/model/#"
-            }
-        ],
+        "wmo:dataPolicy": {
```
Comment (tomkralidis) [inline examples/wcmp2_de.dwd.icon-eps.ALL.json:125]: Should be `"wmo:dataPolicy": "core"`

**Review by tomkralidis — CHANGES_REQUESTED** [review]
Great work @jsieland !  A few minor change requests.

For both the GC and icon JSON files, can you pretty print them?  So:

```bash
cat examples/de.dwd.globalcache.json | python -m json.tool > tmp.file && mv tmp.file examples/de.dwd.globalcache.json
cat examples/wcmp2_de.dwd.icon-eps.ALL.json | python -m json.tool > tmp.file && mv tmp.file examples/wcmp2_de.dwd.icon-eps.ALL.json
```
…then commit the updates?  Thanks!

File: examples/wcmp2_de.dwd.icon-eps.ALL.json (hunk 2 above)
Comment (jsieland) [reply examples/wcmp2_de.dwd.icon-eps.ALL.json]: I knew I forgot something... changed - thanks!

**Pull request merged**